### PR TITLE
Update relationships.md

### PR DESCRIPTION
**Pull request by bartduisters**
Rewrite some of the sentences. Add extra information and examples for clarification.

### CODE REVIEW

File: docs/guides/relationships.md [hunk 1]
```diff
@@ -7,23 +7,43 @@ order: 4
...
 
+Import the `Relationship` field:
+```javascript
+const { Text, CalendarDay, Checkbox, Relationship } = require('@keystonejs/fields');
+```
+
+Old code:
 ```javascript
-{
+// ...
```
Comment (MadeByMike) [inline docs/guides/relationships.md]:
```suggestion
```

File: docs/guides/relationships.md [hunk 2]
```diff
@@ -7,23 +7,43 @@ order: 4
...
+assignee: {
+    type: Text,
+    isRequired: true,
+},
+// ...
```
Comment (MadeByMike) [inline docs/guides/relationships.md]:
```suggestion
```

File: docs/guides/relationships.md [hunk 3]
```diff
@@ -7,23 +7,43 @@ order: 4
...
+```
+New code:
+```javascript
+// ...
```
Comment (MadeByMike) [inline docs/guides/relationships.md]:
```suggestion
```

File: docs/guides/relationships.md [hunk 4]
```diff
@@ -7,23 +7,43 @@ order: 4
...
+assignee: {
     type: Relationship,
     ref: 'User',
-}
+    isRequired: true,
+},
+// ...
```
Comment (MadeByMike) [inline docs/guides/relationships.md]:
```suggestion
```

File: docs/guides/relationships.md [hunk 5]
```diff
@@ -61,11 +81,29 @@ assignee: {
...
+```javascript
+task: {
+    type: Relationship,
+    ref: 'Todo.assignee',
+}
+```
+
+To test it out, remove the `isRequired: true` property from the `assignee` field in the `Todos` file:
+```javascript
+// ...
```
Comment (MadeByMike) [inline docs/guides/relationships.md]:
```suggestion
```

File: docs/guides/relationships.md [hunk 6]
```diff
@@ -61,11 +81,29 @@ assignee: {
...
+assignee: {
+    type: Relationship,
+    ref: 'User.task',
+    // isRequired: true,
+},
+// ...
```
Comment (MadeByMike) [inline docs/guides/relationships.md]:
```suggestion
```

File: docs/guides/relationships.md [hunk 7]
```diff
@@ -61,11 +81,29 @@ assignee: {
...
-Take a look at admin panel and try to assign a Todo to a user. Notice that the user's `task` field is already set! Ensure that this works the other way.
+Start the AdminUI and create a Todo and assign a user. Check the user's `task` field and notice that it is already set! When a user is created and a Todo is assigned, the `assignee` field won't be filled in. Add the following code to make it work both ways:
+```javascript
+task: {
+    type: Relationship,
+    ref: 'Todo.assignee',
+}
+```
+
+To test it out, remove the `isRequired: true` property from the `assignee` field in the `Todos` file:
```
Comment (MadeByMike) [inline docs/guides/relationships.md]:
```suggestion
To test it out, remove the `isRequired: true` property from the `assignee` field in the `Todos` file:

```

Comment (MadeByMike) [reply docs/guides/relationships.md]: Adding a new line before code blocks

File: docs/guides/relationships.md [hunk 8]
```diff
@@ -7,23 +7,43 @@ order: 4
...
-Now, when we've created all necessary lists, let's link them together by setting up
+Let's link the Todo list and the User list together by setting up
 a `relationship`. Tweak the `assignee` field in `Todos.js` to match the following code:
 
+Import the `Relationship` field:
+```javascript
+const { Text, CalendarDay, Checkbox, Relationship } = require('@keystonejs/fields');
+```
+
+Old code:
```
Comment (MadeByMike) [inline docs/guides/relationships.md]:
```suggestion
Old code:

```

File: docs/guides/relationships.md [hunk 9]
```diff
@@ -7,23 +7,43 @@ order: 4
...
 
+This chapter assumes that that the reader has the code that was created in [Setup - Chapter 1](https://www.keystonejs.com/guides/new-project) and [Setup - Chapter 2](https://www.keystonejs.com/guides/add-lists).
+
 ## Pick assignee from Users collection (to-single relationship)
 
-Now, when we've created all necessary lists, let's link them together by setting up
+Let's link the Todo list and the User list together by setting up
 a `relationship`. Tweak the `assignee` field in `Todos.js` to match the following code:
 
+Import the `Relationship` field:
```
Comment (MadeByMike) [inline docs/guides/relationships.md]:
```suggestion
Import the `Relationship` field:

```

File: docs/guides/relationships.md [hunk 10]
```diff
@@ -7,23 +7,43 @@ order: 4
...
 ```javascript
-{
+// ...
+assignee: {
+    type: Text,
+    isRequired: true,
+},
+// ...
+```
+New code:
```
Comment (MadeByMike) [inline docs/guides/relationships.md]:
```suggestion

New code:

```

**Comment by MadeByMike** [comment]
Mostly just formatting suggestions here.

File: docs/guides/relationships.md (hunk 1 above)
Comment (bartduisters) [reply docs/guides/relationships.md]: // ... was used in other places to indicate that there is more code above or below the piece of code, but if the preference is to not have them, fine by me

**Pull request merged**